<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentWidget</class>
 <widget class="QWidget" name="AddStudentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>371</width>
     <height>147</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="idLabel">
        <property name="text">
         <string>学号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sexLabel">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mathLabel">
        <property name="text">
         <string>高等数学 成绩</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cProgramLabel">
        <property name="text">
         <string>程序设计 成绩</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="idEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="sexEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="mathEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="cProgramEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加学生信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>169</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
